From 3045a4d587fc051903cbe7bb771c6c4478ca1096 Mon Sep 17 00:00:00 2001
From: Nikolay Borisov <nborisov@suse.com>
Date: Mon, 19 Jun 2017 12:28:26 +0300
Subject: [PATCH] splice: Stub splice_write_to_file
Patch-mainline: Never, Suse-specific
References: bsc#1043234

This is needed to appease the SLE11-SP4 symset grouping

Signed-off-by: Nikolay Borisov <nborisov@suse.com>
---
 fs/splice.c | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/fs/splice.c b/fs/splice.c
index 372da4f30255..c80f2bc4b58a 100644
--- a/fs/splice.c
+++ b/fs/splice.c
@@ -979,6 +979,20 @@ ssize_t splice_from_pipe(struct pipe_inode_info *pipe, struct file *out,
 	return ret;
 }
 
+typedef ssize_t (splice_write_actor)(struct pipe_inode_info *,
+                                     struct splice_desc *);
+
+
+ssize_t
+splice_write_to_file(struct pipe_inode_info *pipe, struct file *out,
+                        loff_t *ppos, size_t len, unsigned int flags,
+                        splice_write_actor actor)
+{
+	return -EINVAL;
+}
+EXPORT_SYMBOL(splice_write_to_file);
+
+
 /**
  * generic_file_splice_write - splice data from a pipe to a file
  * @pipe:	pipe info
-- 
2.7.4

